deployment/pyproject.toml

```diff
@@ -3,7 +3,7 @@ requires = ["hatchling"]
 build-backend = "hatchling.build"
 
 [project]
-name = "strongmind_deployment"
+name = "strongmind_deployment_beta"
```

Comment:
so you'd just keep deploying from `caylent_main`? If so, then this bit need to be only brought into that branch.

Reply:
right, the branch trigger will detect it.  Was crossing my wires with manual workflows that need to be on main first. 

I have changed the target of this PR.

Reply:
I guess some changes were on the pypackage.toml file on caylent_main so I had to rebase and resolve.